src/api/app/controllers/webui/project_controller.rb

```diff
@@ -61,6 +61,21 @@ def index
     end
   end
 
+  def remove_role
+    authorize @project, :update?
+    super
+  end
+
+  def save_group
+    authorize @project, :update?
+    super
+  end
+
+  def save_person
+    authorize @project, :update?
+    super
+  end
```

Comment:
I don't get it why don't you authorize in `Webui::ManageRelationships`?

Reply:
I'm not doing anything in ```Webui::ManageRelationships``` that`s why I'm "overwriting" the methods in the controller were they are mixed in via ```include``` and calling authorize and super. Test suite was failing because of them and for me was better to do an authorization check there than white-list it.. what do you think @hennevogel ?

Reply:
> I'm not doing anything in Webui::ManageRelationships

Clearly, but why? The `Webui::ManageRelationships` mixin brings controller methods. Why don't you add the authorization into the mixin?

Reply:
Hm, the ```lib``` is as generic as possible. Specific bits for this controller should be done in the controller itself, no? IMO the lib and authorization shouldn't be tight coupled.

Reply:
> Hm, the mixing is as generic as possible. 

Nope